src/Pages/SignUp/SignUpCenterBox/SignUpCenterBox.tsx

```diff
@@ -1,18 +1,74 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import './SignUpCenterBox.scss';
+import { useDispatch, useSelector } from "react-redux";
 
-import {SignUpInputsBox} from './SignUpInputsBox';
-import {SignUpButtonBox} from './SignUpButtonBox';
 import {PageTittle} from "../../../components/PageTittle";
+import { Input } from "../../../components/Input";
+import { Button } from "../../../components/Button";
+import { signUp } from "../../../thunkAction/SignUp";
+import { useNavigate } from "react-router-dom";
+
 
 export const SignUpCenterBox = () => {
+    const dataSignUp = useSelector((state: any) => state.signUp);
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+    
+    const [username, setUsername] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+
+    const onNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setUsername(event.target.value)
+    } 
+    const onEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value)
+    }  
+    const onPasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value)
+    }
+    const onConfirmPasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setConfirmPassword(event.target.value)
+    }
+
+    useEffect(() => {
+        if (dataSignUp.dataSignUp != null) {
```

Comment:
Не используй двойное равно (==, !=) и здесь можно упростить
```suggestion
        if (dataSignUp.dataSignUp) {
```